### PR TITLE
Implements tokens display (all of them) in the clerical tool

### PR DESCRIPTION
**Pull request by Mironor**

### CODE REVIEW

File: parsers/src/main/scala/uk/gov/ons/addressIndex/parsers/Tokens.scala [hunk 1]
```diff
@@ -11,6 +11,7 @@ import scala.util.matching.Regex
   */
 object Tokens extends CrfTokenable {
```
Comment (RichardSmithONS) [inline parsers/src/main/scala/uk/gov/ons/addressIndex/parsers/Tokens.scala]: I have made a similar change to this class in my branch and have included a paoEndSuffix field - is it needed?

File: demo-ui/app/uk/gov/ons/addressIndex/demoui/views/clericalTool.scala.html [hunk 2]
```diff
@@ -154,182 +155,583 @@ <h3 class="green">@aresponse.total @messages("single.found")</h3>
...
+                                            @aresponse.tokens.getOrElse(Tokens.townName, "")
+                                            </div>
+                                            <div class="taColumn4">
+                                            @address.nag.map{ nag => @nag.townName}
                                             </div>
                                         </div>
                                         <div class="taRow">
                                             <div class="taColumn1">
-                                                @messages("clerical.buildingnumber")
+                                            @messages("clerical.token.town_name")
```
Comment (RichardSmithONS) [inline demo-ui/app/uk/gov/ons/addressIndex/demoui/views/clericalTool.scala.html]: Leaving the duplicates token names for now is fine. I wouldn't be happy about converting the divs to a table as this overturns an agreement I had with Alan (there are many discussions on the internet on DIVs v tables for reactiveness etc.). It is not particularly hard to implement the equivalent of rowspan with divs and css

<div style="display: table; width:300px; border: 1px solid black; border-spacing: 2px;">
  <div style="display: table-cell; border: 1px solid black; vertical-align: middle;">
    This column should equal the height (no fixed-height allowed) of the 2 rows sitting to the right.
  </div>
  <div style="display: table-cell; width: 100px;">
    <div style="border: 1px solid black; margin-bottom: 2px;">
      Here is some sample text. And some additional sample text.
    </div>
    <div style="border: 1px solid black;">
      Here is some sample text. And some additional sample text.
    </div>
  </div>
</div>

Comment (RichardSmithONS) [inline demo-ui/app/uk/gov/ons/addressIndex/demoui/views/clericalTool.scala.html]:
```<div style="display: table; width:300px; border: 1px solid black; border-spacing: 2px;">
  <div style="display: table-cell; border: 1px solid black; vertical-align: middle;">
    This column should equal the height (no fixed-height allowed) of the 2 rows sitting to the right.
  </div>
  <div style="display: table-cell; width: 100px;">
    <div style="border: 1px solid black; margin-bottom: 2px;">
      Here is some sample text. And some additional sample text.
    </div>
    <div style="border: 1px solid black;">
      Here is some sample text. And some additional sample text.
    </div>
  </div>
</div>
```

File: parsers/src/main/scala/uk/gov/ons/addressIndex/parsers/Tokens.scala (hunk 1 above)
Comment (Mironor) [inline parsers/src/main/scala/uk/gov/ons/addressIndex/parsers/Tokens.scala]: It will be merged eventually when your branch is merged with develop.

**Pull request merged**